Mlem/Views/Shared/Comments/Components/CollapsedCommentReplies.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CollapsedCommentReplies.swift
+//  Mlem
+//
+//  Created by Sumeet Gill on 2024-01-04.
+//
+
+import SwiftUI
+
+struct CollapsedCommentReplies: View {
+    @Binding var numberOfReplies: Int
+    var lineWidth: CGFloat = 2
+    
+    var body: some View {
+        let replyText = numberOfReplies != 1 ? "comments" : "comment"
+
+        HStack {
+            Rectangle()
+                .border(width: lineWidth, edges: [.leading], color: .accentColor)
+                .frame(width: lineWidth)
+            Image(systemName: Icons.replies)
+            Text("show \(numberOfReplies) \(replyText)")
+                .foregroundStyle(.blue)
+        }
+        .padding(.top, 5)
+        .padding(.bottom, 10)
+        .frame(maxHeight: 50)
+    }
```

Comment:
What's your opinion on these small design tweaks for the "show n replies" button? 

<img src="https://github.com/mlemgroup/mlem/assets/78750526/75199937-c268-4213-b790-5735d8d063ef" width=60%>


```suggestion
        HStack {
            Rectangle()
                .border(width: lineWidth, edges: [.leading], color: .accentColor)
                .frame(width: lineWidth)
            Image(systemName: Icons.replies)
            Text("Show ^[\(numberOfReplies) Reply](inflect: true)")
                .foregroundStyle(.blue)
                .padding(.vertical, 10)
        }
        .frame(maxHeight: 50)
        .padding(.leading, 10)
```